charts/k8s-integration/values.yaml

```diff
@@ -0,0 +1,45 @@
+global:
+  imagePullSecrets:
```

Comment:
For what use is the image pull secrets if we specify in every sub chart a new image pull secrets?

Reply:
The motivation is to allow the customer to configure it only once instead of multiple times per component.